content/applications/inventory_and_mrp/plm/manage_changes/eco_type.rst

```diff
@@ -84,51 +109,40 @@ filling it in, click the :guilabel:`Add` button to finish adding the stage.
    Adding another stage helps the product manager track unassigned tasks.
 
    .. image:: eco_type/create-stage.png
-      :align: center
       :alt: Create a new stage in a project for an ECO type.
 
-Verification stage
-------------------
-
-Click an ECO type from :menuselection:`PLM app --> Overview` to open a kanban view of |ECOs| of this
-type.
+Use case: configuring verification and closing stages
+-----------------------------------------------------
 
-To configure a verification stage, hover over the intended stage, and select the :guilabel:`⚙️
-(gear)` icon. Then, click :guilabel:`Edit` to open a pop-up window.
+Verification stage
+~~~~~~~~~~~~~~~~~~
```

Comment:
Disagree with the heading changes here
this isn't a use-case, it's a series of explicit instructions